schematics/dashboard/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, <% if(!!viewEncapsulation) { %>, ViewEncapsulation<% }%><% if(changeDetection !== 'Default') { %>, ChangeDetectionStrategy<% }%> } from '@angular/core';
+
+declare var google: any;
+
+@Component({
+  selector: '<%= selector %>',<% if(inlineTemplate) { %>
+  template: `
+    <div class="grid-container">
+      <h1 class="mat-h1">Dashboard</h1>
+      <mat-grid-list cols="2" rowHeight="350px">
+        <mat-grid-tile *ngFor="let card of cards">
+          <mat-card>
+            <mat-card-header>
+              <mat-card-title>
+                {{card.title}}
+                <button mat-icon-button [matMenuTriggerFor]="menu">
+                  <mat-icon aria-label="Menu icon">more_vert</mat-icon>
```

Comment:
Same note as the other template. The `aria-label` should be on the button and be more descriptive.